src/conn/conn_api.c

```diff
@@ -550,6 +550,9 @@ __conn_add_extractor(
     WT_NAMED_EXTRACTOR *nextractor;
     WT_SESSION_IMPL *session;
 
+    if (wt_conn != NULL)
```

Comment:
I suspect this is temporary?

Reply:
Oops - thanks for spotting that. Fixed in 22326a0.